core/src/test/scala/org/apache/spark/scheduler/TaskResultGetterSuite.scala

```diff
@@ -81,6 +81,16 @@ class TaskResultGetterSuite extends SparkFunSuite with BeforeAndAfter with Local
   // as we can make it) so the tests don't take too long.
   def conf: SparkConf = new SparkConf().set("spark.akka.frameSize", "1")
 
+  test("Kryo serializer for TaskResult") {
+    val conf1 = new SparkConf(false)
+    conf1.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
```

Comment:
I wasn't aware of SPARK-12414 previously.
If Java serializer is to be used everywhere, is there some code in this PR that is useful for SPARK-12414 ?